Marlin/GCodes.h

```diff
@@ -0,0 +1,47 @@
+/**
+ * GCodes.h - Contains codes for Gcodes.
+ * Author: Dan Thompson (danthompson41@gmail.com)
+ * Date: 4/8/2016
+ * Used as the home for Gcodes, to decouple them from Marlin_main. 
+ * Note on the weirdness of having the definitions in an h file: This is to
+ * facilitate the inline functions. Inline is only file specific, so doesn't
+ * work if you count on the linker (ex, using the normal .cpp with an h file
+ * interface). Directly including the code in an H file is the easiest way to
+ * do this.
+ * Copyright (C) 2016 Voxel8
+ */
+
+#ifndef G_CODES_H_
+#define G_CODES_H_
+
+#include "Marlin.h"
+#include "GCodeUtility.h"
+#include "planner.h"
+
+//===========================================================================
+//============================= Public Functions ============================
+//===========================================================================
+
+inline void gcode_M272(void) {
+  for (int8_t i = 0; i < NUM_AXIS; i++) {
+    if (code_seen(axis_codes[i])) {
+      if (i == E_AXIS) {
+        float value = (213.3333333 / (1 + (code_value() / 1000)));
```

Comment:
It would be good to get some resolution on these numbers. I'm fairly certain that 1000 is mm to um. Do we have a define for that somewhere?
At least we should comment in something like "conversion from ball-bar units to um"